src/Donors/Endpoints/ListDonors.php

```diff
@@ -181,7 +182,9 @@ public function getDonors(): array
      */
     public function getTotalDonorsCount(): int
     {
-        $query = give()->donors->prepareQuery();
+        $builder = new ModelQueryBuilder(Donor::class);
```

Comment:
Same comment as before about not using the `ModelQueryBuilder`

Reply:
Resolved: e13dd57ce3cc1184463bf52711077f0c3b212542